docroot/themes/humsci/humsci_basic/humsci_basic.libraries.yml

```diff
@@ -1,6 +1 @@
 base:
```

Comment:
Don't leave remnants of a library. either remove the file entirely, or define the library.